src/ensembl/.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     '@typescript-eslint/interface-name-prefix': 0,
     '@typescript-eslint/prefer-interface': 0,
     '@typescript-eslint/explicit-function-return-type': 0,
+    "@typescript-eslint/explicit-module-boundary-types": 0,
```

Comment:
Out of curiosity: do you remember the place where eslint complained otherwise?

Reply:
This started happening recently. It was complaining about no explicit return type for default exports.